pkg/vault/contracts/Vault.sol

```diff
@@ -1221,8 +1224,8 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
                 );
```

Comment:
Partially outside the diff. Comment above:

// In a wrap operation, the underlying balance of the buffer will decrease and the wrapped balance will
// increase. To decrease the underlying balance, we get the delta amount that was deposited
// (vaultUnderlyingDelta) and discount the amount needed for the wrapping operation
// (amountInUnderlying). The same logic applies to wrapped balances.